src/middlewares/phoenix/actions/index.js

```diff
@@ -201,8 +201,11 @@ export function connectToPhoenixChannelForEvents({
             channel,
           })
         );
-        dispatch(endPhoenixChannelProgress({ channelTopic, loadingStatusKey: channelTopic }));
+        dispatch(
+          endPhoenixChannelProgress({ dispatch, channelTopic, loadingStatusKey: channelTopic })
+        );
```

Comment:
Please do not use fixed for formatting commits. Formatting is a separated logic that should be added as a separate commit.